packages/core/src/renderers/renderer.util.tsx

```diff
@@ -54,31 +56,26 @@ export const mapStateToLayoutProps = (state, ownProps) => {
   return {
     renderers: state.renderers,
     visible,
-    path: ownProps.path
+    path: ownProps.path,
+    getStyle: getStyle(state),
```

Comment:
this is vanilla renderer specific, maybe this should be handled similar to the vanillacontrolprops ?